src/state/global/notifications/notificationMachine.ts

```diff
@@ -0,0 +1,67 @@
+import { v4 } from 'uuid';
+import { ContextFrom, EventFrom } from 'xstate';
+import { createModel } from 'xstate/lib/model';
+
+export type Severity = 'information' | 'warning' | 'error' | 'success';
+
+export type Notification = {
+  id: string;
+  severity: Severity;
+  message: string;
+};
+
+// export type NotificationEvent = {
```

Comment:
That is nice. Was there an issue using it?

Reply:
reverted